.github/ISSUE_TEMPLATE.md

```diff
@@ -1,36 +1,21 @@
 <!--
-- Please ask questions at https://groups.google.com/forum/#!forum/loopbackjs or
-  https://gitter.im/strongloop/loopback
+- Please ask questions at https://groups.google.com/forum/#!forum/loopbackjs
+  or https://gitter.im/strongloop/loopback
 
-- Immediate support is available through our subscription plans, see
-  https://strongloop.com/api-connect-faqs/
+- Immediate support is available through our subscription plans,
+  see https://strongloop.com/api-connect-faqs/
 -->
 
-### Bug or feature request
-
-<!--
-Mark your choice with an "x" (eg. [x], NOT [*]).
--->
-
-- [ ] Bug
-- [ ] Feature request
-
-### Description of feature (or steps to reproduce if bug)
-
-
-
-### Link to sample repo to reproduce issue (if bug)
-
+### Bug/Feature request
 
+<!-- Description of feature
+or steps to reproduce/link to sample repo if bug -->
 
 ### Expected result
 
+<!-- Include actual results if bug -->
 
+### Additional information
```

Comment:
It would be nice to get the output of `node -e 'console.log(process.platform, process.arch,process.versions.node)'`. For example: `darwin x64 6.9.2`.